### ISSUE TITLE
Fetching Data for Pages - Implement the Post Page - Q/A problem

### ISSUE THREAD

**Issue by metasean** [arunoda/learnnextjs-content#30]
On the https://learnnextjs.com/basics/fetching-data-for-pages/fetching-data-for-pages lesson, the question at the end asks whether the login message was seen on terminal, browser, both, or neither.  On my machine, no matter what I do, I only see the message in the terminal (and then I only see it after a 404 and a very bloody long wait - _between 3-15 *seconds*_), but the "correct answer" says I should only be seeing it in the browser.

https://github.com/arunoda/learnnextjs-content/blob/master/content/1-basics/7-fetching-data-for-pages.js#L160

```
  "dependencies": {
    "express": "^4.15.3",
    "isomorphic-unfetch": "^1.0.0",
    "next": "^2.4.4",
    "react": "^15.6.1",
    "react-dom": "^15.6.1"
  }
```

**Comment by metasean** [comment]
Here's a repo with the code - https://github.com/metasean-toys/next-js-tut/tree/next-tutorial.  
On the `next-tutorial` branch, run `npm run dev` to see what's happening.

**Comment by rouxfeur** [comment]
I had the same problem. Fixed by using Next ^4.2.0 instead of ^5.0.0.
